datasets/usda-cdl/collection/description.md

```diff
@@ -0,0 +1 @@
+The USDA NASS Cropland Data Layers (CDLs) include Cropland, Confidence, Cultivated, and Frequency products. The Cropland product is a crop-specific land cover data layer created annually for the contiguous United States (CONUS). A corresponding Confidence product represents the predicted confidence that is associated with an output pixel. A value of zero indicates low confidence, while a value of 100 indicates high confidence. The Cultivated product identifies cultivated and non-cultivated land cover for CONUS and is based on land cover information derived from the 2017 through 2021 Cropland products. The Frequency product identifies crop specific planting frequency and are based on land cover information derived from the 2008 through 2021 Cropland products. There are currently four individual crop frequency data layers that represent four major crops: corn, cotton, soybeans, and wheat. All products have a 30 meter resolution.
```

Comment:
https://www.usgs.gov/centers/fort-collins-science-center/science/usda-national-agricultural-statistics-service-cropland seems to be a nice overview. Can we copy paste that for the first paragraph (with a link to the source)?

> The Cropland Data Layer (CDL) is a product of the USDA National Agricultural Statistics Service (NASS) with the mission “to provide timely, accurate and useful statistics in service to U.S. agriculture” (Johnson and Mueller, 2010, p. 1204). The CDL is a crop-specific land cover classification product of more than 100 crop categories grown in the United States. CDLs are derived using a supervised land cover classification of satellite imagery. The supervised classification relies on first manually identifying pixels within certain images, often called training sites, which represent the same crop or land cover type. Using these training sites, a spectral signature is developed for each crop type that is then used by the analysis software to identify all other pixels in the satellite image representing the same crop. Using this method, a new CDL is compiled annually and released to the public a few months after the end of the growing season.

Then something like,

> This collection includes Cropland, Confidence, Cultivated, and Frequency products.
> * Cropland: Crop-specific land cover data created annually. There are currently four individual crop frequency data layers that represent four major crops: corn, cotton, soybeans, and wheat.
> * Confidence: The predicted confidence associated with an output pixel. A value of zero indicates low confidence, while a value of 100 indicates high confidence.
> * Cultivated: cultivated and non-cultivated land cover for CONUS based on land cover information derived from the 2017 through 2021 Cropland products.
> * Frequency: crop specific planting frequency based on land cover information derived from the 2008 through 2021 Cropland products.